src/components/organisms/PublicFooter.vue

```diff
@@ -0,0 +1,92 @@
+<script setup>
+import { computed } from 'vue'
+import { APP_TITLE } from '~/constants'
+import DcLocalePicker from '../molecules/DcLocalePicker.vue';
+import { useI18n } from 'vue-i18n';
+
+const { t } = useI18n()
+
+const menu = computed(() => [
```

Comment:
About the typings, I might have commented in the wrong line, sorry. 
But here for instance we could have a type like an Array of MenuItem